traits/examples/introduction/5_documentation.py

```diff
@@ -67,7 +67,8 @@ class Image(HasStrictTraits):
 then if we type::
 
     >>> image.fileanme = "sample_0002.png"
-    TraitError: Cannot set the undefined 'filename' attribute of a 'Image' object.
+    TraitError: Cannot set the undefined 'fileanme' attribute of a 'Image'
```

Comment:
Yes, we're deliberately introducing a typo here, so that the error message matches reality. This a deliberate demonstration of `HasStrictTraits`'s ability to catch typos.